### ISSUE TITLE
[Request]: Make tuba buildable on Debian stable

### ISSUE THREAD

**Issue by msdemlei** [GeopJr/Tuba#422]
### Describe the request

I'd like to avoid flatpak and friends, and it seems there's not *terribly* much missing; a ``make`` on bookworm seems to combile a large bunch of sources and fails with an apparently modest:

```
../src/Dialogs/ProfileEdit.vala:418.11-418.22: error: The name `ResponseType' does not exist in the context of `Tuba.Dialogs.ProfileEdit.choose_file._lambda118_'
  418 |                     case ResponseType.ACCEPT:
      |                          ^~~~~~~~~~~~
../src/Application.vala:122.5-122.13: error: The name `GtkSource' does not exist in the context of `Tuba.Application.startup'
  122 |                 GtkSource.init ();
      |                 ^~~~~~~~~
Compilation failed: 2 error(s), 0 warning(s)
```

I freely admit that I understand zero vala, and I can't see where the GtkSource (or ResponseType) symbol would otherwise come from; but it *looks* as if fixing this might not be overwhelmingly much work.

Is it?  Where would ResponseType and GtkSource come from if the dependencies have the right versions?

### Implementation Details

- [ ] This should be an option in settings.
- [ ] This should be only available to some fediverse backends. (Include which ones on the above field).
- [X] This is client-only (and shouldn't sync with the instance).
- [X] This follows the [GNOME HIG](https://developer.gnome.org/hig/).

**Comment by GeopJr** [comment]
Thanks for raising this issue!

It should be fixed as of ea725cec5a871cc5713b9541d4fcefc2caa2a6eb, 91ec3d55d13b2d85af00ee65fdfcf471f33563db

Keep in mind that while I've been trying to keep compatibility with GTK < 4.10 (or even 4.8) and for building without GtkSourceView, any release after GNOME 45 won't include them as the new libadwaita widgets require GTK >= 4.11.3 and glib >= 2.76.0

**Comment by msdemlei** [comment]
On Mon, Jul 31, 2023 at 02:20:54PM -0700, Evan Paterakis wrote:
> It should be fixed as of ea725cec5a871cc5713b9541d4fcefc2caa2a6eb, 91ec3d55d13b2d85af00ee65fdfcf471f33563db

Yup, works for me.  Thanks a lot!
